tests/Integrations/GoogleSpanner/GoogleSpannerIntegrationTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Tests\Integration;
+
+use DDTrace\Integrations\GoogleSpanner\GoogleSpannerIntegration;
+use DDTrace\Tag;
+use Google\Cloud\Spanner\SpannerClient;
+use PHPUnit\Framework\TestCase;
+
+class GoogleSpannerIntegrationTest extends TestCase
```

Comment:
When moving away from mocking, this class will have to extend `DDTrace\Tests\Common\IntegrationTestCase` and ideally make use of `isolateTracerSnapshot` ([Guzzle Example](https://github.com/DataDog/dd-trace-php/blob/352d3babf9588fcfbc9726232f0a2594b17dede8/tests/Integrations/Guzzle/V7/GuzzleIntegrationTest.php#L58-L84)), so that snapshot traces are generated under the `tests/snapshots/` directory :+1: